src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$calendar-day-with: 100px;
```

Comment:
There seems to be a typo in the variable name `$calendar-day-with`. It should probably be `$calendar-day-width` to accurately describe its purpose as the width of a calendar day.